docs/conf.py

```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = 'minfraud'
-copyright = '2015-2016, MaxMind, Inc'
+copyright = '2015-2017, MaxMind, Inc'
```

Comment:
Is there any reason to not bump to 2018 if we're doing this?

Reply:
The only reason is that I have a hard time keeping track of what year it is.